lib/tokenize.cpp

```diff
@@ -2024,9 +2036,26 @@ bool Tokenizer::simplifyUsing()
                         continue;
 
                     // remove the qualification
-                    while (tok1->strAt(-1) == "::" && tok1->strAt(-2) == scope) {
-                        tok1->deletePrevious();
-                        tok1->deletePrevious();
+                    std::string fullScope = scope;
```

Comment:
It seems fullScope could be const?

Reply:
It gets changed on line 2055:` fullScope.resize(idx - 3);`

Reply:
ok